### PR TITLE
More cache improvements.

### PR DESCRIPTION
**Pull request by jezdez**

### CODE REVIEW

**Comment by jezdez** [comment]
We'll have to increase the `KUMASCRIPT_MAX_AGE` constance value in production manually from 5 seconds to something more sensible like the default of 600.

File: kuma/wiki/kumascript.py [hunk 1]
```diff
@@ -153,66 +152,68 @@ def get(document, cache_control, base_url, timeout=None):
             files=files,
             attachments=files,  # Just for sake of verbiage?
             slug=document.slug,
-            tags=[x.name for x in document.tags.all()],
-            review_tags=[x.name for x in
-                         document.current_revision.review_tags.all()],
+            tags=list(document.tags.values_list('name', flat=True)),
+            review_tags=list(document.current_revision
+                                     .review_tags
+                                     .values_list('name', flat=True)),
```
Comment (robhudson) [inline kuma/wiki/kumascript.py]: Nice improvement and it should be faster too. :+1:

File: kuma/wiki/kumascript.py [hunk 2]
```diff
@@ -225,58 +226,60 @@ def add_env_headers(headers, env_vars):
...
-        fl_packets = defaultdict(dict)
-        for k, v in response.headers.items():
-            if not k.lower().startswith('firelogger-'):
+        packets = defaultdict(dict)
+        for key, value in response.headers.items():
+            if not key.lower().startswith('firelogger-'):
                 continue
-            _, packet_id, seq = k.split('-', 3)
-            fl_packets[packet_id][seq] = v
+            _, id_, seq = key.split('-', 3)
```
Comment (robhudson) [inline kuma/wiki/kumascript.py]: I try to avoid using `_` b/c of how often people import gettext as `_`.

**Comment by robhudson** [comment]
r+wc

File: kuma/wiki/kumascript.py (hunk 2 above)
Comment (jezdez) [inline kuma/wiki/kumascript.py]: Good point, I'll fix

File: kuma/wiki/kumascript.py (hunk 1 above)
Comment (jezdez) [inline kuma/wiki/kumascript.py]: :dancers:

**Pull request merged**